### PR TITLE
Add option to make water opaque

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2-ui/windows/TopToolbar.cpp [hunk 1]
```diff
@@ -3626,38 +3629,47 @@ static void top_toolbar_init_view_menu(rct_window* w, rct_widget* widget)
         ToggleOption(DDIDX_VIEW_CLIPPING, STR_VIEW_CLIPPING_MENU),
         ToggleOption(DDIDX_HIGHLIGHT_PATH_ISSUES, STR_HIGHLIGHT_PATH_ISSUES_MENU),
     };
+
     static_assert(ItemIDsMatchIndices(items));
 
     SetItems(items);
 
+    // Opaque water relies on RCT1 sprites.
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/TopToolbar.cpp:3637]: This block needs to be moved down because it won't work otherwise.

**Comment by AaronVanGeffen** [comment]
Nice work! This takes me back to #613.

I get why you're calling the option 'Opaque water'. However, from a player perspective, I think it would be nicer to instead offer the option 'Transparent water' or 'Underwater view', either being checked by default. This would also be more in line with the option offered by the original RCT games, particularly if it is called 'Underwater view'.

**Comment by Gymnasiast** [comment]
OK, did that. Also added a shortcut for it (defaulting to 2, like in RCT1).

File: distribution/changelog.txt [hunk 2]
```diff
@@ -6,6 +6,7 @@
 - Feature: [#14538] [Plugin] Add property for getting current plugin api version.
 - Feature: [#14620] [Plugin] Add properties related to guest generation.
 - Feature: [#14636] [Plugin] Add properties related to climate and weather.
+- Feature: [#14731] Opaque water (like in RCT1).
```
Comment (tupaschoal) [inline distribution/changelog.txt:9]: Should this have reflected the "Transparent water" string?

Comment (Gymnasiast) [reply distribution/changelog.txt:9]: Nah, don’t think so. Transparent water is what we already had, opaque is what we added.

**Pull request merged**